### PR TITLE
pypi-release git tags automation

### PR DESCRIPTION
**Pull request by cumason123**
Builds onto PR #87 

## Changes
Added extra steps to [pypi-release.yml](.github/workflows/pypi-release.yml) which will create git tags on successful pypi version releases when pushing to master.

Created [pypi_packaging.py](pypi_packaging.py) which contains a `pypi_config` object and a method to create git tags depending on PyPI version specified within `pypi_config`

## One line description for the changelog
pypi-release git tags automation

- [*] Tests pass
- [*] Appropriate changes to README are included in PR

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -40,5 +41,5 @@
     ],
     package_dir={"": "cloudevents"},
     packages=["http", "sdk"],
-    version="1.0.0",
+    version=pypi_config["version_target"],
```
Comment (di) [inline setup.py:44]: I think I'd prefer getting these from a more generic location, like `cloudevents.__version__`.

See some examples here: https://packaging.python.org/guides/single-sourcing-package-version/

File: .github/workflows/pypi-release.yml [hunk 2]
```diff
@@ -22,3 +22,7 @@ jobs:
         uses: pypa/gh-action-pypi-publish@master
         with:
           password: ${{ secrets.pypi_password }}
+      - name: Install cloudevents and GitPython
+        run: pip install -U cloudevents 'GitPython==3.1.7'
```
Comment (di) [inline .github/workflows/pypi-release.yml:26]:
```suggestion
        run: pip install -r requirements/publish.txt
```
Why do we need to install `cloudevents` here?

File: setup.py (hunk 1 above)
Comment (cumason123) [reply setup.py:44]: working on it

File: .github/workflows/pypi-release.yml (hunk 2 above)
Comment (cumason123) [reply .github/workflows/pypi-release.yml]: Yeah. the git tag creation wants an installation of the most updated version of cloudevents to test if the version was successfully pushed to pypi before making the git tag.

**Pull request merged**